scripts/vm/network/vnet/modifyvxlan.sh

```diff
@@ -131,6 +131,9 @@ fi
 
 LOCKFILE=/var/run/cloud/vxlan.lock
 
+# ensures that parent directories exists and prepares the lock file
+mkdir -p "${LOCKFILE%/*}" && touch "$LOCKFILE"
```

Comment:
we do not need the touch right?

Reply:
Wouldn't actually require the file creation